plugins/repository-azure/src/test/java/org/elasticsearch/cloud/azure/storage/AzureStorageServiceTests.java

```diff
@@ -143,6 +146,18 @@ public void testGetSelectedClientNoTimeout() {
         assertThat(client1.getDefaultRequestOptions().getTimeoutIntervalInMs(), is(nullValue()));
     }
 
+    public void testGetSelectedClientBackoffPolicy() {
```

Comment:
I did not add a test initially for that because I can't test that the retry value is the one I set.
But I agree. So I'm adding a test which just set:

```
        Settings timeoutSettings = Settings.builder()
            .put("cloud.azure.storage.azure.account", "myaccount")
            .put("cloud.azure.storage.azure.key", "mykey")
            .put("cloud.azure.storage.azure.max_retries", 7)
            .build();
```

And make sure that it does not fail.